ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ViewVulnReportPage.tsx

```diff
@@ -1,7 +1,227 @@
-import React from 'react';
+import React, { ReactElement, useEffect, useState } from 'react';
+import { useHistory, useParams, Link, generatePath } from 'react-router-dom';
+import {
+    PageSection,
+    Title,
+    Divider,
+    Flex,
+    FlexItem,
+    Breadcrumb,
+    BreadcrumbItem,
+    Bullseye,
+    Spinner,
+    Dropdown,
+    DropdownToggle,
+    DropdownItem,
+    DropdownSeparator,
+    Tabs,
+    Tab,
+    TabTitleText,
+    TabTitleIcon,
+} from '@patternfly/react-core';
+import { CaretDownIcon, DownloadIcon, HistoryIcon, HomeIcon } from '@patternfly/react-icons';
+
+import { vulnerabilityReportPath, vulnerabilityReportsPath } from 'routePaths';
+import useFetchReport from 'Containers/Vulnerabilities/VulnerablityReporting/api/useFetchReport';
+
+import PageTitle from 'Components/PageTitle';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import NotFoundMessage from 'Components/NotFoundMessage/NotFoundMessage';
+import useDeleteModal from '../hooks/useDeleteModal';
+import DeleteReportModal from '../components/DeleteReportModal';
+
+export type TabTitleProps = {
+    icon?: ReactElement;
+    children: string;
+};
+
+function TabTitle({ icon, children }: TabTitleProps): ReactElement {
+    return (
+        <Flex alignItems={{ default: 'alignItemsCenter' }}>
+            {icon && (
+                <FlexItem>
+                    <TabTitleIcon>{icon}</TabTitleIcon>
+                </FlexItem>
+            )}
+            <FlexItem>
+                <TabTitleText>{children}</TabTitleText>
+            </FlexItem>
+        </Flex>
+    );
+}
 
 function ViewVulnReportPage() {
-    return <div />;
+    const history = useHistory();
+    const { reportId } = useParams();
+    const [isActionsDropdownOpen, setIsActionsDropdownOpen] = useState(false);
+
+    const { reportConfiguration, isLoading, error } = useFetchReport(reportId);
+
+    const {
+        openDeleteModal,
+        isDeleteModalOpen,
+        closeDeleteModal,
+        isDeleting,
+        onDelete,
+        deleteError,
+        isDeleted,
+    } = useDeleteModal();
```

Comment:
Similar to my other question about `useFetchReport`, I noticed you are also adding `useDeleteModal` in the page above this. 

Could you call this once and pass the object returned by `useDeleteModal` down to this view?

Reply:
Same as my previous comment